src/canopy_bioemi_mod.F90

```diff
@@ -143,6 +165,12 @@ SUBROUTINE CANOPY_BIO( ZK, FCLAI, FCH, LAI, FSUN, PPFD_SUN, &
             GAMMACO2 = 1.0_rk
         end if
 
+! Get LEAF AGE factor
+        TABOVECANOPY  = TLEAF_AVE  !Assume instantaneous TLEAF estimate for TABOVECANOPY
```

Comment:
I think you should just use the input 2-m temperature from model/obs, and it should not be indexed on canopy levels (ZK).